pkg/cache/v3/user.go

```diff
@@ -92,7 +92,11 @@ func (u *userAttributesCache) Put(userAttributes *userproto.UserAttributes) erro
 		for _, value := range attribute.Values {
 			pipe.SAdd(key, value)
 		}
-		pipe.Expire(key, userAttributeTTL)
+		setTTL := defalutUserAttributeTTL
+		if ttl > 0 {
+			setTTL = ttl
+		}
+		pipe.Expire(key, time.Duration(setTTL)*time.Second)
```

Comment:
We don't need the default value because if the user wants to keep the keys permanently, they will set it to zero.
Also, we already set the default value in the config.

```suggestion
		pipe.Expire(key, time.Duration(ttl)*time.Second)
```